deployment/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: Deployment
+---
+
+# Deployment
+
+These sections will discuss options for a production deployment of the OCS applications, as well as information on intial setup for the main OCS applications. To begin, go to [Local Deployment]({% link deployment/local_deployment.md %}) for instructions on how to run the OCS applications on a single machine or network of machines, or go to [Cloud Deployment]({% link deployment/cloud_deployment.md %}) for information on deploying the OCS applications to a cloud service provider such as AWS in kubernetes with helm charts.
```

Comment:
I had the kubernetes w/helm charts statement because I figured that is what we would provide for the "cloud" section. I guess maybe it's more correct to just call it a kubernetes deployment section? If its just using normal machines in AWS to run the docker-compose files then they can just use the local deployment section.

Reply:
I've just renamed the cloud deployment to be kubernetes deployment since I think that fits better with what we want to describe there.

Reply:
Sounds good 👍